bcipy/gui/BCInterface.py

```diff
@@ -327,6 +328,54 @@ def check_input(self) -> bool:
             return False
         return True
 
+    def user_id_validator(self) -> bool:
+        """Check User ID Requirements. User ID must follow the following Requirements:
+
+        1. Maximum length of 25 letters or numbers
```

Comment:
We could add these as class-level variables at the top. For instance,

`self.max_length = 25` in the init

and then use that value instead of the hardcoded 25. Update the documentation to reflect that we set it above. This will make changing it easier in the future. I'd do this for all cases. `self.min_length`, `self.special_character_regex`